packages/embark-plugin-cmd/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "embark-plugin-cmd",
+  "version": "4.1.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Plugin installation utility for Embark",
+  "homepage": "https://github.com/embark-framework/embark/tree/master/packages/embark-plugin-cmd#readme",
+  "bugs": "https://github.com/embark-framework/embark/issues",
+  "license": "MIT",
+  "repository": {
+    "directory": "packages/embark-plugin-cmd",
+    "type": "git",
+    "url": "https://github.com/embark-framework/embark.git"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "embark-utils": "^4.1.0-beta.0",
+    "rimraf": "2.6.3"
```

Comment:
This package doesn't seem to use `rimraf`. However it uses `async` which is missing here in the dependencies.